examples/create-env-cluster-topics/main.tf

```diff
@@ -114,6 +120,22 @@ module "honest_labs_kafka_topic_example_2" {
   depends_on                   = [module.cluster_admin_privilege_service_account]
 }
 
+resource "confluent_kafka_acl" "kafka_acl_consumer" {
```

Comment:
we took this resource out of the module intentionally - why are we putting it back here?